### PR TITLE
A tiny-js-world

### PR DESCRIPTION
**Pull request by IhorOzerov**
### A tiny js world

[Demo](https://ihorozerov.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/IhorOzerov/a-tiny-JS-world/blob/master/index.js)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by IhorOzerov** [comment]
I have completed self-checks and fixed the code accordingly!

File: submissions/IhorOzerov/a-tiny-JS-world/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+const extraWords = {
+   creature: "I am ",
+   name: "My name is ",
+   gender: "I am a pretty ",
+   hands: "I have a ",
+   legs: "hands and ",
+   friends: "legs I like to talk with ",
+   say: "I would like to say "
+}
```
Comment (OleksiyRudenko) [inline submissions/IhorOzerov/a-tiny-JS-world/index.js:53]: Great solution to add property explanations.

File: submissions/IhorOzerov/a-tiny-JS-world/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+   say: "Where is my money, Gregory?!",
+   friends:  ["Gregory", "Vanessa"]
+};
+const catWoman = Object.create(cat);
+catWoman.creature = "cat-woman";
+catWoman.name = "Vanessa";
+catWoman.gender = 'female';
+catWoman.legs = 2;
+catWoman.hands = 2;
+catWoman.friends =  ["Yohanna", "Gangster"];
```
Comment (OleksiyRudenko) [inline submissions/IhorOzerov/a-tiny-JS-world/index.js:43]: You do not need to fix anything here. Just to reflect on when you re-approach TJSW after a while (OOP Exercise).
Would it be more efficient to extend CatWoman from Woman (or Human) and hard-bind only saying with Cat?

**Review by OleksiyRudenko — APPROVED** [review]
@IhorOzerov great job!

File: submissions/IhorOzerov/a-tiny-JS-world/index.js (hunk 1 above)
Comment (IhorOzerov) [reply submissions/IhorOzerov/a-tiny-JS-world/index.js:53]: > Great solution to add property explanations.

It was your advice, thank you.

**Pull request merged**